README.md

```diff
@@ -124,11 +128,15 @@ This is an example docs mapping:
 
 ```
 .
-├──page-1.md
-├──page-2.md
+├──page-1
+│   └──+page.md
+├──page-2
+│   └──+page.md
 └──category
-    ├──category-page-1.md
-    └──category-page-2.md
+    ├──category-page-1
+    │   └──+page.md
+    └──category-page-2
+        └──+page.md
```

Comment:
Why do the markdown files need to be named +[name].md (same for the svelte files).

Reply:
Basically, because of how SvelteKit's new router works, the route name is now determined by the folder it's in, not the file names. So, instead of using `docs/article.md` we use `docs/article/+page.md`.

Reply:
This is required by SvelteKit to mark a route-related file.